includes/admin/home/tasks/class-sensei-home-tasks-provider.php

```diff
@@ -85,4 +88,12 @@ private function map_task( Sensei_Home_Task $task ): array {
 		];
 	}
 
+	/**
+	 * Mark tasks list as completed.
+	 *
+	 * @param bool $completed Whether the task list must be marked as completed or uncompleted.
+	 */
+	public function mark_as_completed( $completed = true ) {
+		update_option( self::COMPLETED_TASKS_OPTION_KEY, $completed, false );
```

Comment:
Is there any special reason to set the last argument (option autoload) to `false`?

Reply:
I'm not Aarón, but I believe the reasoning behind this is to avoid loading the option in **every** request. In other words, that option would load ONLY when it's requested using `get_option`, which I believe it makes sense.

At least, that's what I understand based [on the documentation](https://developer.wordpress.org/reference/functions/update_option/#parameters).

Reply:
@fjorgemota described the reason quite well. Also I added it basically because @jom suggested it to me and it made sense.